File: SearchInsertPosition/SearchInsertPosition.py
# Runtime: 52 ms, faster than 70.79% of Python3 online submissions for Search Insert Position.
# Memory Usage: 14.5 MB, less than 72.42% of Python3 online submissions for Search Insert Position.

class Solution:
    def searchInsert(self, nums, target):
        return len([i for i in nums if i < target])

File: SearchInsertPosition/SearchInsertPosition.cpp
/*
Runtime: 8 ms, faster than 93.32% of C++ online submissions for Search Insert Position.
Memory Usage: 9.7 MB, less than 54.32% of C++ online submissions for Search Insert Position.
*/

class Solution {
public:
    int searchInsert(vector<int>& nums, int target) {
        int i = 0;
        while (nums[i] < target) {
            i ++;
            if (i == nums.size()) break;     
        }
        return i;
    } 
};